Instagram/Views/PostCell.m

```diff
@@ -11,13 +11,10 @@ @implementation PostCell
 
 - (void)awakeFromNib {
```

Comment:
So unless you do something else in here, the whole thing is pointless. It will call the super class if there isn't one implemented here. Swift forces you to type the keyword `override` to make it more clear what's going on but ObjC it's a little harder to realize that's what's happening.